backend/lib/index.js

```diff
@@ -14,7 +15,10 @@ const version = require("./endpoints/version");
 
 module.exports = function createApp(config) {
   const app = fastify({
-    logger: true,
+    logger: {
+      level: config.logger.level,
+      stream: logStream(config.logger),
```

Comment:
maybe this should only create a stream if we want a log stream in config?
Because now it seems like if you want to only configure the log level, it will always create a log stream
I already saw a stream write error in Sentry logs, might be related to this?

Reply:
This [error](https://sentry.io/organizations/fightpandemics/issues/1708382915/?project=5251967&query=is%3Aunresolved)? This was me testing locally; I forgot to remove the SENTRY_DSN environment variable from my .env so it sent to Sentry.

I'll change this to conditionally add the stream.